app/components/admin/error_summary_component.rb

```diff
@@ -51,4 +51,34 @@ def errors
                   object.errors
                 end
   end
+
+  def error_message_text(error)
+    if custom_error_present_in_locales?(error)
+      custom_error_message(error)
```

Comment:
i actually think this should just be `error.message` tbh

I ran into an issue with this being overwritten by [this](https://github.com/alphagov/whitehall/blob/8b7b867216ec45a3c15b6d00db5507187882a41c/app/models/date_validation.rb#L59) custom error message.

I think the right thing to do though is add custom messages for each affected attribute in the locales files.